test/plugin/test_in_tail.rb

```diff
@@ -323,6 +350,69 @@ def test_emit_with_read_lines_limit(data)
       assert num_events <= d.emit_count
     end
 
+    sub_test_case "log throttling per file" do
+      teardown do
+        FileUtils.rm_f("#{TMP_DIR}/tail.txt")
```

Comment:
Should use `cleanup_file`

Reply:
Fixed in https://github.com/fluent/fluentd/pull/3185/commits/5fdabd3f6238d0f5524dcf966d94c84583f8f36b.